src/components/password-input/index.md

```diff
@@ -0,0 +1,155 @@
+<div class="app-wcag-22" id="wcag-copy-paste-password" role="note">
+  {{ govukTag({
+    text: "WCAG 2.2",
+    classes: "app-tag"
+  }) }}
+  <p>You must allow users to paste or autofill their password. Avoid making the user memorise or transcribe a password from somewhere else in order to use your service. This is to comply with WCAG 2.2 success criterion <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum">3.3.8 Accessible Authentication (Minimum)</a>.</p>
+</div>
+
+Users will not be able to copy any content from the password field when it’s set to ‘hide’ password, such as to save a password in a password manager. However, copying is possible when the field is set to ‘show’.
+
+These are features of browser behaviour and cannot be overridden.
+
+### Avoid restricting the user's input
+
+See the [Ask users for Passwords](/patterns/passwords/) pattern to see how to help users choose strong passwords.
+
+Support all the characters users may need to enter a password, including numbers and symbols.
+
+If you must place password restrictions on users, such as for technical or security reasons, be clear and consistent.
+
+Any restrictions must be identical wherever the user creates or enters a password. If you change the restrictions over time, you must continue to support existing user passwords or ask them to set a new one.
+
+#### Do not use 'maxlength' to restrict password length
+
+Users will not get any feedback when they’ve reached the `maxlength` and their text input has been truncated. This is particularly true when text has been pasted from elsewhere or autofilled by a password manager.
+
+If you must restrict the length of a password, show an error message instead using the [validation](/patterns/validation/) pattern.
+
+### Do not spellcheck or autocapitalise the user's input
+
+Some browsers might automatically change what the user is typing when the input’s text is visible, such as correcting spelling or automatically turning on upper case letters at the start of sentences.
+
+You can tell browsers not to correct spellings by setting the `spellcheck` attribute to `false`.
+
+Doing this can also prevent ['spell-jacking'](https://www.otto-js.com/news/article/chrome-and-edge-enhanced-spellcheck-features-expose-pii-even-your-passwords), where security researchers have found some spell checking tools gathering personal identifiable information, even user’s passwords, from password input fields to send to third party services.
+
+You can tell browsers not to autocapitalise values by setting the `autocapitalize` attribute to `off`.
+
+## Research on this component
+
+Some browsers, password managers and similar tools will add their own show and hide functionality within the password input field. This might confuse users shown alongside this component’s show and hide button.
+
+We investigated how this component interacts with the similar functionality and worked to prevent this duplication where possible.
```

Comment:
Draft content only – assumptions is that we'll use this space to summarise conclusions from #3552